plugins/providers/virtualbox/action.rb

```diff
@@ -58,7 +58,7 @@ module Action
 
       # This action boots the VM, assuming the VM is in a state that requires
       # a bootup (i.e. not saved).
-      def self.action_boot
+      def self.action_boot(cloud_init=false)
```

Comment:
Small nitpick on these, but I think it'll be more maintainable if this is something like `**opts` instead, just in case more options like this are ever introduced.